app/assets/javascripts/controllers/order_summary_controller.coffee

```diff
@@ -0,0 +1,18 @@
+app = angular.module('customerApp')
+
+app.controller 'OrderSummaryController',['$scope', 'Order', ($scope, Order) ->
+  Order.getPlaced().$promise.then (data) ->
```

Comment:
you shouldn't need a specific endpoint to fetch this. the Order#index is already an endpoint that lists the orders, you just need to pass parameters so that you can filter the data in the index function